test-support/page-object/properties/value.js

```diff
@@ -1,42 +1,31 @@
-import Descriptor from '../descriptor';
 import { findElementWithAssert, trim } from '../helpers';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["trim" is defined but never used](https://www.codacy.com/app/san650/ember-cli-page-object/file/1940277127/issues/source?bid=2903707&fileBranchId=3041068#l1)